OIPA/iati/models.py

```diff
@@ -283,15 +296,6 @@ def __unicode__(self,):
         return "%s - %s" % (self.code, self.name)
 
 
-class TransactionType(models.Model):
```

Comment:
Nice Job on cleaning up the models.py.
How sure are you however, that moving these classes (Transaction & TransactionType) does not mess up some of the untested modules in oipa, like the old API and the parser?